LICENSE.txt

```diff
@@ -1,202 +1,16 @@
+Apache Software License 2.0
```

Comment:
The cruft template may have the wrong license text which is not recognised by GitHub. See PR in Hummingbird:
https://github.com/bird-house/hummingbird/pull/40

Reply:
Done and fix in cookiecutter template as well.